installexporter.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/bash
+
+git clone https://github.com/metalkube/ironic-prometheus-exporter/
```

Comment:
Any plans to package this for RDO?

Reply:
We haven't consider this in any meetings or stories, maybe in the next sprint or something I would say.

Reply:
Let's add a story for it. We cannot use `git clone` forever.